tsdb/engine/tsm1/reader.go

```diff
@@ -236,6 +238,18 @@ func (t *TSMReader) Close() error {
 	return t.accessor.close()
 }
 
+func (t *TSMReader) ValidateChecksum() map[string]map[string]bool {
```

Comment:
This shouldn't be on the `TSMReader`.  It's too expensive of a call.  Can you move this to the `influx_inspect` too that handles verification?